games-puzzle/quadra/quadra-1.3.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools desktop
+
+DESCRIPTION="Tetris clone with multiplayer support"
+HOMEPAGE="https://github.com/quadra-game/quadra"
+SRC_URI="https://github.com/quadra-game/quadra/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-libs/boost:=
+	media-libs/libpng:=
+	media-libs/libsdl2[opengl,sound,video]
```

Comment:
> However, turns out not `X` is required but `opengl`

I don't think it does, I have -opengl by default on my libsdl2 to notice these, and just tried again and it runs fine with -opengl. From a quick look at the source, I don't see any reference to GL either.

But ultimately it can't work with /just/ minimal flags and will still need either X or wayland, but as I said we typically don't check that /at least one/ backend is selected for neither audio nor video (need USE=wayland to work on wayland, USE=X to work on X, USE=pulseaudio to use pulse, etc...). If that's ever checked it'd be libsdl2 that should with a REQUIRED_USE if USE=video rather than this ebuild anyhow.

> BTW, it does run on wayland too (running on sway with libsdl2 build without X).

Nice :)

Reply:
Thanks for the detailed explanation - I think understand now. 
I've removed the `opengl` flag again.